modules/equativBidAdapter.js

```diff
@@ -152,22 +131,17 @@ export const converter = ortbConverter({
     const bid = context.bidRequests[0];
     const req = buildRequest(imps, bidderRequest, context);
 
-    if (deepAccess(bid, 'ortb2.site.publisher')) {
-      deepSetValue(req, 'site.publisher.id', bid.ortb2.site.publisher.id || bid.params.networkId);
-    } else if (deepAccess(bid, 'ortb2.app.publisher')) {
-      deepSetValue(req, 'app.publisher.id', bid.ortb2.app.publisher.id || bid.params.networkId);
-    } else if (deepAccess(bid, 'ortb2.dooh.publisher')) {
-      deepSetValue(req, 'dooh.publisher.id', bid.ortb2.dooh.publisher.id || bid.params.networkId);
-    } else {
-      deepSetValue(req, 'site.publisher.id', bid.params.networkId);
-    }
+    let env = ['ortb2.site.publisher', 'ortb2.app.publisher', 'ortb2.dooh.publisher'].find(env => deepAccess(bid, env)) || 'ortb2.site.publisher';
```

Comment:
I like this code compression!

The one thing I found initially confusing was the double use of `env` (after both the `let` declaration and also within the `find()` method).

Would it be more readable if the second `env` was renamed, maybe to something like `propPath`?

```
    let env = ['ortb2.site.publisher', 'ortb2.app.publisher', 'ortb2.dooh.publisher'].find(propPath => deepAccess(bid, propPath)) || 'ortb2.site.publisher';

```

I like the use of array methods here and with `mimes` and `placement` later.  Really tidies things up!

Reply:
Indeed, it's confusing. Updated. Thx.